Sources/Extensions/Watch/ComplicationController.swift

```diff
@@ -78,8 +78,15 @@ class ComplicationController: NSObject, CLKComplicationDataSource {
         if let fromComplication = self.template(for: complication) {
             template = fromComplication
         } else {
-            Current.Log.error("failed to get template, providing error")
-            template = ComplicationGroupMember(family: complication.family).errorTemplate
+            Current.Log.info("no configured template, providing placeholder")
```

Comment:
Do these remain available when a complication is configured for that family?

Reply:
on watchOS 7, yes. on watchOS 6, no.